src/components/Footer/Footer.js

```diff
@@ -1,7 +1,70 @@
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faFacebook,
+  faInstagram,
+  faYoutube,
+  faAndroid,
+  faApple,
+} from '@fortawesome/free-brands-svg-icons';
+import './Footer.scss';
 
 const Footer = () => {
-  return <div></div>;
+  return (
+    <footer>
+      <div className="footer">
```

Comment:
```suggestion
      <footer className="footer">
```
무의미하게 deps를 깊게 하는 형태라
하나로 합쳐도 좋을 것 같습니다.

Reply:
네, 푸시 하기 전 다시 한 번 더 체크하여 무의미한 형태는 넣지 않도록 하겠습니다.